concreteproperties/post.py

```diff
@@ -14,6 +14,7 @@ def plotting_context(
     ax: Optional[matplotlib.axes.Axes] = None,  # type: ignore
     pause: bool = True,
     title: str = "",
+    aspect: bool = False,
```

Comment:
Are you able to add a simple description of this argument in the docstring under the ``title`` description?

E.g. :param aspect: blah blah

Reply:
@robbievanleeuwen, should be resolved now provided the plots in docs are ok.